cmd/root.go

```diff
@@ -52,7 +52,8 @@ func Execute(ver string) {
 
 func init() {
 	// Global flags
-	rootCmd.PersistentFlags().StringVarP(&ctx.From, "from", "F", "", "A continent, region (e.g eastern europe), country, US state or city (default \"world\")")
+	rootCmd.PersistentFlags().StringVarP(&ctx.From, "from", "F", "", `Comma-separated list of "magic" location values to match against. For example the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network (default "world").
+You can read more about "magic" location values here: https://github.com/jsdelivr/globalping/blob/master/docs/measurement/schema/location.md#magic-query`)
```

Comment:
We shouldn't link to outside docs because that URL will change soon. Lets remove that part and if needed just expand the description

Reply:
Sure, I removed the link now, the currently provided text should be enough